.travis.yml

```diff
@@ -1,3 +1,10 @@
+# Fix Travis errors when sending large output to STDOUT. These write errors occurred after
+# a Travis update to new Trusty images on Dec. 12th 2017. The reason for these
+# write errors is unknown. Using the deprecated builds did not fix the problem.
+# Setting 'filter_secrets: false' as suggested here
+# https://github.com/travis-ci/travis-ci/issues/4704#issuecomment-321777557
+# fixes the problem.
+filter_secrets: false
 language: python
```

Comment:
I am glad it works now but I feel uneasy about not finding the effect of this parameter in any doc, mostly regarding secrets such as our oAuth token that allows pushing to github release...